### PR TITLE
Recommendation feature built 

### PR DESCRIPTION
**Pull request by PrettyPearl2302**
## Description
In this pull request, I build my recommendation feature in the following steps: 

1. After seeding my rating table with random rating values, I created a route in _server.js_ to fetch all the episodes that were rated greater than or equal to 4 from the database, for a user with a specific id. With this fetch, I also included the genre of podcasts the episodes belonged to. 
2. I created a route in my second server, _routes.js_ to fetch new podcasts from the API with 'Genre' as a required parameter. This was the crucial part in building the recommendations. Because Taddy provides a way to filter search results by genre, I used the searchForTerm which I had previously described in _app.js_  to fetch the podcasts, and planned to pass the genres from the response in _server.js_ as the genre that the responses would be filtered with. 
3. On the front end in _Recommendation.jsx_, I fetched the response data from my new route in _server.js_ and then I iterated over each genre, then passed it as the query parameter in the second fetch I made in _Recommendation.jsx_ to get the podcast data from my API-fetching route in routes.js, "/api/recommendations"

From there, I went over a similar process like I did with _Home.jsx_ to display the results to the user. I passed the data from _Recommendation.jsx_ to a podcast grid component, which would pass to a podcast card component. 

Issues I still have is that somehow the data that I passed to _Recgrid.jsx_ is for only one array of podcast data that the _fetchPodcastsByRec_  function returns. 

I also have to handle an edge case where _fetchRatingsbyValue_ returns genres that are similar, and I don't have to pass every iteration if some are identical. 

## Milestones
TECHNICAL CHALLENGE #2: Recommendations

## Test Plan

https://github.com/PrettyPearl2302/timify_capstone/assets/116607525/dba000ef-6844-49e1-a885-22493116395f

### CODE REVIEW

File: timify-ui/src/components/Home/Home.jsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ const Home = () => {
       try {
         const response = await fetch('http://localhost:5000/api/home');
         const data = await response.json();
+        console.log(data)
```
Comment (jeffwdeng) [inline timify-ui/src/components/Home/Home.jsx:18]: need this?

**Review by jeffwdeng — APPROVED** [review]

File: timify-ui/src/components/Home/Home.jsx (hunk 1 above)
Comment (PrettyPearl2302) [reply timify-ui/src/components/Home/Home.jsx:18]: i just took it out in my next pr
